mlir/lib/ExecutionEngine/ROCm/rocm-runtime-wrappers.cpp

```diff
@@ -286,6 +333,12 @@ extern "C" void mgpuMemCopy2DFloat(float *sourceAllocated, float *sourceAligned,
 
 // 4D float memref utility routines.
 
+short tensorValue(short min, short max) {
```

Comment:
the function name doesn't seem to mean it returns a random number.

Reply:
I've renamed this function to "randomValue()"